.gitignore

```diff
@@ -0,0 +1,51 @@
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+*.pyc
+
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Logs and databases #
+######################
+*.log
+*.sql
+*.sqlite
+
+# LaTeX Generated Files #
```

Comment:
I will admit to just including a boilerplate .gitignore that has everything
that I usually don't want to push. Perhaps it isn't the most concise, but I
don't think individuals will have issue with it and could modify to fit
specific use as needed.

Or I could just change it for the pull request :)

On Tue, May 17, 2016 at 5:10 PM Will Benton <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/willb/wi-thesis-template/pull/3#discussion_r63613389>:
>
> > +*.7z
> > +*.dmg
> > +*.gz
> > +*.iso
> > +*.jar
> > +*.rar
> > +*.tar
> > +*.zip
> > +
> > +# Logs and databases #
> > +######################
> > +*.log
> > +*.sql
> > +*.sqlite
> > +
> > +# LaTeX Generated Files #
>
> It would be much better for this to only exclude temporary files generated
> by LaTeX. (For example, someone with a sql or sqlite file used to store
> experiment results and automatically generate figures might be surprised to
> find it excluded from version control.)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/willb/wi-thesis-template/pull/3/files/b9852d04476cd951b43e969e4ba8bbc75b1b6fb1..642934b69daba76b188d9e1687c3895e09617080#r63613389>
>